From 0e984106a89ac1732a5200f7b3180454a6c3ad15 Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Sat, 31 Dec 2011 01:07:49 +0100
Subject: [PATCH] socket: rename the PassCred= option to PassCredentials=,
 since we don't want to needlessly abbreviate options unless
 they are very well established (cherry picked from commit
 271b032a053f9d4a1be271bb052276ae27fe36c6)

Conflicts:

	units/systemd-journald.socket
---
 man/systemd.socket.xml           |    8 ++++----
 src/dbus-socket.c                |    4 ++--
 src/load-fragment-gperf.gperf.m4 |    2 +-
 src/socket.c                     |    2 +-
 units/syslog.socket              |    2 +-
 units/systemd-shutdownd.socket   |    2 +-
 6 files changed, 10 insertions(+), 10 deletions(-)

diff --git a/man/systemd.socket.xml b/man/systemd.socket.xml
index f883543..ef5b28c 100644
--- a/man/systemd.socket.xml
+++ b/man/systemd.socket.xml
@@ -510,7 +510,7 @@
                                 <term><varname>Transparent=</varname></term>
                                 <listitem><para>Takes a boolean
                                 value. Controls the IP_TRANSPARENT
-                                option. Defaults to
+                                socket option. Defaults to
                                 <option>false</option>.</para></listitem>
                         </varlistentry>
 
@@ -518,17 +518,17 @@
                                 <term><varname>Broadcast=</varname></term>
                                 <listitem><para>Takes a boolean
                                 value. This controls the SO_BROADCAST
-                                option, which allows broadcast
+                                socket option, which allows broadcast
                                 datagrams to be sent from this
                                 socket. Defaults to
                                 <option>false</option>.</para></listitem>
                         </varlistentry>
 
                         <varlistentry>
-                                <term><varname>PassCred=</varname></term>
+                                <term><varname>PassCredentials=</varname></term>
                                 <listitem><para>Takes a boolean
                                 value. This controls the SO_PASSCRED
-                                option, which allows UNIX sockets to
+                                socket option, which allows UNIX sockets to
                                 receive the credentials of the sending
                                 process in an ancillary message.
                                 Defaults to
diff --git a/src/dbus-socket.c b/src/dbus-socket.c
index 37ab7eb..c428189 100644
--- a/src/dbus-socket.c
+++ b/src/dbus-socket.c
@@ -51,7 +51,7 @@
         "  <property name=\"FreeBind\" type=\"b\" access=\"read\"/>\n"  \
         "  <property name=\"Transparent\" type=\"b\" access=\"read\"/>\n" \
         "  <property name=\"Broadcast\" type=\"b\" access=\"read\"/>\n" \
-        "  <property name=\"PassCred\" type=\"b\" access=\"read\"/>\n" \
+        "  <property name=\"PassCredentials\" type=\"b\" access=\"read\"/>\n" \
         "  <property name=\"Mark\" type=\"i\" access=\"read\"/>\n"      \
         "  <property name=\"MaxConnections\" type=\"u\" access=\"read\"/>\n" \
         "  <property name=\"NAccepted\" type=\"u\" access=\"read\"/>\n" \
@@ -114,7 +114,7 @@ DBusHandlerResult bus_socket_message_handler(Unit *u, DBusConnection *c, DBusMes
                 { "org.freedesktop.systemd1.Socket", "FreeBind",       bus_property_append_bool,         "b", &u->socket.free_bind       },
                 { "org.freedesktop.systemd1.Socket", "Transparent",    bus_property_append_bool,         "b", &u->socket.transparent     },
                 { "org.freedesktop.systemd1.Socket", "Broadcast",      bus_property_append_bool,         "b", &u->socket.broadcast       },
-                { "org.freedesktop.systemd1.Socket", "PassCred",       bus_property_append_bool,         "b", &u->socket.pass_cred       },
+                { "org.freedesktop.systemd1.Socket", "PassCredentials",bus_property_append_bool,         "b", &u->socket.pass_cred       },
                 { "org.freedesktop.systemd1.Socket", "Mark",           bus_property_append_int,          "i", &u->socket.mark            },
                 { "org.freedesktop.systemd1.Socket", "MaxConnections", bus_property_append_unsigned,     "u", &u->socket.max_connections },
                 { "org.freedesktop.systemd1.Socket", "NConnections",   bus_property_append_unsigned,     "u", &u->socket.n_connections   },
diff --git a/src/load-fragment-gperf.gperf.m4 b/src/load-fragment-gperf.gperf.m4
index 35ec005..81e186c 100644
--- a/src/load-fragment-gperf.gperf.m4
+++ b/src/load-fragment-gperf.gperf.m4
@@ -177,7 +177,7 @@ Socket.PipeSize,                 config_parse_size,                  0,
 Socket.FreeBind,                 config_parse_bool,                  0,                             offsetof(Socket, free_bind)
 Socket.Transparent,              config_parse_bool,                  0,                             offsetof(Socket, transparent)
 Socket.Broadcast,                config_parse_bool,                  0,                             offsetof(Socket, broadcast)
-Socket.PassCred,                 config_parse_bool,                  0,                             offsetof(Socket, pass_cred)
+Socket.PassCredentials,          config_parse_bool,                  0,                             offsetof(Socket, pass_cred)
 Socket.TCPCongestion,            config_parse_string,                0,                             offsetof(Socket, tcp_congestion)
 Socket.MessageQueueMaxMessages,  config_parse_long,                  0,                             offsetof(Socket, mq_maxmsg)
 Socket.MessageQueueMessageSize,  config_parse_long,                  0,                             offsetof(Socket, mq_msgsize)
diff --git a/src/socket.c b/src/socket.c
index 0864cce..bbfc842 100644
--- a/src/socket.c
+++ b/src/socket.c
@@ -406,7 +406,7 @@ static void socket_dump(Unit *u, FILE *f, const char *prefix) {
                 "%sFreeBind: %s\n"
                 "%sTransparent: %s\n"
                 "%sBroadcast: %s\n"
-                "%sPassCred: %s\n"
+                "%sPassCrededentials: %s\n"
                 "%sTCPCongestion: %s\n",
                 prefix, socket_state_to_string(s->state),
                 prefix, socket_address_bind_ipv6_only_to_string(s->bind_ipv6_only),
diff --git a/units/syslog.socket b/units/syslog.socket
index e74b559..ccf79fe 100644
--- a/units/syslog.socket
+++ b/units/syslog.socket
@@ -18,7 +18,7 @@ Wants=syslog.target
 [Socket]
 ListenDatagram=/dev/log
 SocketMode=0666
-PassCred=yes
+PassCredentials=yes
 
 # The service we activate on incoming traffic is
 # systemd-kmsg-syslogd.service. That doesn't mean however, that this
diff --git a/units/systemd-shutdownd.socket b/units/systemd-shutdownd.socket
index 13b6c7a..532a6f0 100644
--- a/units/systemd-shutdownd.socket
+++ b/units/systemd-shutdownd.socket
@@ -15,4 +15,4 @@ Before=sockets.target
 [Socket]
 ListenDatagram=/run/systemd/shutdownd
 SocketMode=0600
-PassCred=yes
+PassCredentials=yes
-- 
1.7.7.5

